Makefile

```diff
@@ -74,7 +74,7 @@ docker:
 	docker cp ethermint:/usr/bin/emintcli ./build/
 
 docker-localnet:
-	# build the image
+# build the image
 	docker build -f ./networks/local/ethermintnode/Dockerfile . -t emintd/node
```

Comment:
i dont know if moving this comment will break other OS but for windows if the comment isnt moved over it tries to run it.